packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -17,6 +17,7 @@
 import * as api from '@opentelemetry/api';
 import { Context, HrTime, SpanAttributeValue } from '@opentelemetry/api';
```

Comment:
This should fix the build. :slightly_smiling_face: 
```suggestion
import {
  Context,
  HrTime,
  SpanAttributeValue,
  SpanContext,
  SpanKind,
  Link,
  TimeInput
} from '@opentelemetry/api';
```